weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1603,24 +1603,31 @@ def _query_stream(
 
         summary = None
         parameters = _process_parameters(parameters)
-        with self.ch_client.query_rows_stream(
-            query,
-            parameters=parameters,
-            column_formats=column_formats,
-            use_none=True,
-            settings=settings,
-        ) as stream:
-            if isinstance(stream.source, QueryResult):
-                summary = stream.source.summary
-            logger.info(
-                "clickhouse_stream_query",
-                extra={
-                    "query": query,
-                    "parameters": parameters,
-                    "summary": summary,
-                },
+        try:
```

Comment:
how does this manifest in the trace server? Should we do something on the server side such that this actually shows a 500?